NetCasbin/ThrowHelper.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System;
+using System.Diagnostics;
+
+namespace NetCasbin
+{
+    internal static class ThrowHelper
+    {
+        internal static void ThrowNameNotFoundException()
+            => throw new ArgumentException("The name parameter does not exist.");
+
+        internal static void ThrowOneOfNamesNotFoundException()
+            => throw new ArgumentException("The name1 or name2 parameter does not exist.");
+
+        internal static void ThrowInvalidFieldValuesException()
```

Comment:
Not used code.

Reply:
Ok, I will add this change in another PR.